modules/core/src/main/scala/lucuma/odb/api/repo/ObservationRepo.scala

```diff
@@ -154,5 +165,51 @@ object ObservationRepo {
           Tables.constraintSetObservation,
           ConstraintSetModel.ConstraintSetEvent.updated
         )
+
+      override def setSubject(
+        oids:    List[Observation.Id],
+        subject: Option[Either[Asterism.Id, Target.Id]]
+      ): F[List[ObservationModel]] = {
```

Comment:
This is a bit extra embarrassing.  I'll try to simplify before I mark it for final review.